src/platforms/dart/configuration/integrations/logging.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Logging Integration
+caseStyle: canonical
+supportLevel: production
+sidebar_order: 3
+---
+
+The `sentry_logging` library provides [Logging](https://pub.dev/packages/logging) support for Sentry via [onRecord property](https://pub.dev/documentation/logging/latest/logging/Logger/onRecord.html) that sends events and breadcrumbs to Sentry. Once this integration is configured you can use Logging’s public API.
```

Comment:
```suggestion
The `sentry_logging` library provides [Logging](https://pub.dev/packages/logging) support for Sentry via [onRecord property](https://pub.dev/documentation/logging/latest/logging/Logger/onRecord.html). It is able to collect breadcrumbs and capture events. Once this integration is configured you can use Logging’s public API exclusively or in combination to the Sentry's SDK API to capture and enrich events.
```

Reply:
IMO, this update seems better in that it provides more information. If you two decide to keep it and the content applies to Timber, then I'd make the update there as well. The edits I made before still apply (avoiding Latin words and the same missing comma).

Reply:
Thanks, I will merge both suggestions.